lib/chunks.js

```diff
@@ -968,12 +968,6 @@ module.exports.ensureChunksForCourseAsync = async (courseId, chunks) => {
   // in the results of `select_metadata_for_chunks`. If it does not, we remove
   // the chunk from the course's runtime directory.
   //
-  // At this time, we only do this for `elements`, `elementExtensions`,
-  // `serverFilesCourse`, and `clientFilesCourse`, since it's critical that they
-  // not be present on disk if they're removed from a course. If it becomes
-  // necessary to handle additional chunks in the future, this code can easily
-  // be extended to support that.
-  //
```

Comment:
I'm not sure why I added this comment, as I can't find anywhere that we're actually limiting to these chunks. Interestingly, I think that if we were limiting, things would work correctly, as the listed chunks don't depend on any of the joined columns that I changed in this PR.

Reply:
I also can't see anywhere we actually implement this restriction. Perhaps this was originally intended as an optimization, because it doesn't actually matter if we have old question chunks on disk (for example). However, I think it's best to keep it like it currently is (and remove this comment).

Reply:
I figured out that this refers to the fact that we were only ever deleting chunks from the database, not that we had special-cased deletion from disk.

Reply:
Ah!